sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/IntervalMetricReader.java

```diff
@@ -159,6 +159,11 @@ protected Builder fromConfigMap(
       }
       return this;
     }
+
+    // Visible for testing
+    long getExportIntervalMillis() {
```

Comment:
I like to use a prefix such as `test_` for methods that are not only visible for testing but exist solely for testing.

Reply:
I don't think I've ever done that but don't mind it either. Will wait for another vote and rename.

Reply:
No strong opinion here either. I just think it would be nice to mark the method as "exists only for testing" vs "visibility extended for testing".